docs/proposals/service-binding-specification/implementing-specification.md

```diff
@@ -0,0 +1,129 @@
+aims to create a mechanism that is widely applicable. The benefit of a Kubernetes-wide service binding
+specification is that all of the actors in an ecosystem, including application developers and service
+providers, can work towards a clearly defined abstraction at the edge of their expertise and depend on
+other parties to complete the chain.
+
+On the other hand, the Service Binding Operator has a well-defined mechanism for binding applications
+and services together. This mechanism is different from how the Service Binding Specification for
+Kubernetes lays out binding between entities. Generally, most features defined in the specification
+are a subset of features defined in the Service Binding Operator.
+
+There are major differences between the specification and the current mechanism the Service Binding
+Operator employs such as how the application is defined in the CRD, how a binding secret is
+projected in an application container and etc. The [Service Binding feature comparison document](./service-binding-feature-comparison.pdf) lists the differences between the two.
+
+One of the major gaps between the CRDs defined by the two approaches is that the Service Binding
+Operator supports defining multiple services within a CR. Whereas the specification allows defining
+one service per CR. Here are two main reasons why the specification is in favor of defining one
+service per CR:
+
+- It is easier for tools created around the service binding to add and remove services by simply
+  adding or removing independent CRs instead of having to deal with editing an array of services
+  within a CR. In addition, editing the service array in a CR might cause CR to be corrupted unless
+  concurrent updates are well synchronized across a tool or tools. However, by defining a single
+  service per CR, adding or removing services simply becomes `kubectl apply ...` and
+  `kubectl delete ...`.
+
+- Defining one service per CR allows a cleaner association of attributes with the service. For example,\
+  when we identify a set of extra mappings or environment variables in a `ServiceBinding`, which
+  service does that get associated with if we have multiple services? Or if we want to change the
+  service name or type that gets mounted, we again run into the issue that they all require a way
+  to identify the service you want to apply those attributes against.
+
+Therefore, allowing users of the Service Binding Operator to choose between the specification and the
+current mechanism is likely to increase the user base of the operator, having a more vibrant community
+of service providers.
+
+### Goals
+
+Goals for this proposal are listed as follows:
+
+- Allow users of the Service Binding Operator to have a clear separation in choosing between the
+specification and the current mechanism for binding applications to services.
+
+### Non-Goals
+
+- Implementation details about how the specification should be implemented.
```

Comment:
Some details would be nice, especially how the existing implementation maps to the one proposed. https://github.com/redhat-developer/service-binding-operator/pull/674/files#diff-07b3f77fd05f37abf09d5a741a039c4b 

I think this is a good start
https://github.com/redhat-developer/service-binding-operator/pull/674/files#diff-07b3f77fd05f37abf09d5a741a039c4b